doc/api/n-api.md

```diff
@@ -3216,6 +3216,35 @@ callback invocation, even if it has been successfully cancelled.
 
 ## Version Management
 
+### napi_get_node_version
+<!-- YAML
+added: REPLACEME
+-->
+
+```C
+typedef struct {
+  uint32_t major;
+  uint32_t minor;
+  uint32_t patch;
+  const char* release;
+} napi_node_version;
```

Comment:
Well, eventually one of the goals should be for Node.js itself to use N-API internally, in which case this method would *be* the way to get at this information from JS :-) .. but that's down the road.